gtsfm/scene_optimizer.py

```diff
@@ -400,6 +388,8 @@ def save_full_frontend_metrics(
     """
     metrics_list = []
 
+    round_and_save_fn = lambda x: round(x, PRINT_NUM_SIG_FIGS) if x else None
```

Comment:
Maybe just `round_fn`? This function doesn't explicitly save anything.

Reply:
agreed